dropshot/src/from_map.rs

```diff
@@ -461,6 +463,7 @@ mod test {
         struct B {
             bstring: String,
             boption: Option<String>,
+            // bbool: bool,
```

Comment:
nit: was this commented-out bit intentional?

Reply:
It was intentional. I can't do it now because of serde-rs/serde#1183 -- I can leave it out; up to you.